osu.Game/Online/Rooms/MultiplayerScore.cs

```diff
@@ -13,7 +13,7 @@
 using osu.Game.Scoring;
 using osu.Game.Users;
 
-namespace osu.Game.Online.Multiplayer
+namespace osu.Game.Online.Rooms
```

Comment:
Would propose renaming the class to `RoomScore` too, along with its derivatives (`IndexedMultiplayerScores`, `MultiplayerScore`, `MultiplayerScores`, `MultiplayerScoresAround`) - or maybe moving to the `Multiplayer` namespace? Not sure but something doesn't fit too well here.

Not to mention that we have an `IndexPlaylistScoresRequest` that returns `IndexedMultiplayerScores`... maybe that should be left as a follow up, then? idk